docs/dag-run.rst

```diff
@@ -201,7 +201,7 @@ In addition, you can also manually trigger a DAG Run using the web UI (tab **
 Passing Parameters when triggering dags
 ------------------------------------------
 
-When triggering a DAG from the CLI, the REST API or the UI, it is possible to pass configuration for a DAGRun as
+When triggering a DAG from the CLI, the REST API or the UI, it is possible to pass configuration for a DAG Run as
```

Comment:
```suggestion
When triggering a DAG from the CLI, the REST API or the UI, it is possible to pass configuration for a DagRun as
```

We can cross-link it to DagRun object: https://github.com/apache/airflow/blob/master/airflow/models/dagrun.py#L44